extensions/amp-analytics/0.1/vendors/ibeatanalytics.js

```diff
@@ -19,38 +19,38 @@ import {jsonLiteral} from '../../../../src/json';
 const IBEATANALYTICS_CONFIG = jsonLiteral({
   'requests': {
     'host': 'https://ibeat.indiatimes.com',
-    'base': 'https://ibeat.indiatimes.com/iBeat/pageTrendlogAmp.html',
-    'pageview':
-      '${base}?' +
-      '&h=${h}' +
-      '&d=${h}' +
-      '&url=${url}' +
-      '&k=${key}' +
-      '&ts=${time}' +
-      '&ch=${channel}' +
-      '&sid=${uid}' +
-      '&at=${agentType}' +
+    'track':
+      '${host}/ping-amp?url=${ampdocUrl}' +
+      '&sid=${clientId(_iibeat_session)}' +
+      '&ua=${userAgent}' +
       '&ref=${documentReferrer}' +
+      '&at=${incrementalEngagedTime}' +
+      '&tt=${totalTime}' +
+      '&pid=${pageViewId64}' +
+      '&d=${d}' +
+      '&ct=${ct}' +
+      '&pt=${pt}' +
+      '&au=${au}' +
+      '&ag=${ag}' +
       '&aid=${aid}' +
-      '&loc=1' +
-      '&ct=1' +
-      '&cat=${cat}' +
-      '&scat=${scat}' +
-      '&ac=1' +
-      '&tg=${tags}' +
-      '&ctids=${catIds}' +
-      '&pts=${pagePublishTime}' +
-      '&auth=${author}' +
-      '&pos=${position}' +
-      '&iBeatField=${ibeatFields}' +
-      '&cid=${clientId(MSCSAuthDetails)}',
+      '&cn=${canonicalUrl}' +
+      '&ctIds=${ctIds}',
   },
   'triggers': {
-    'defaultPageview': {
-      'on': 'visible',
-      'request': 'pageview',
+    'trackInterval': {
+      'on': 'timer',
+      'timerSpec': {
+        'interval': 15,
+        'maxTimerLength': 7200,
+      },
+      'request': 'track',
     },
   },
+  'transport': {
```

Comment:
Just to confirm that your server only handles get requests. AMP handles the fallback to xhrpost and image if the browser doesn't support one transportation method. So we'd recommend using the default transport method if that's supported by server. Thanks.

Reply:
@micajuine-ho  removed transport block.